cmd/cyclonus/pkg/cli/analyze.go

```diff
@@ -23,7 +22,6 @@ import (
 const (
 	ParseMode        = "parse"
 	ExplainMode      = "explain"
-	LintMode         = "lint"
```

Comment:
Removing Lint mode since it had compile issues with new structures. This mode flagged Pods (Targets) with NetworkPolicies that fail certain checks e.g. all ingress denied or NetPol doesn’t define the protocol (defaulting to TCP))